webpack.config.js

```diff
@@ -22,7 +22,20 @@ module.exports = {
             loader: 'babel-loader',
             options: {
               cacheDirectory: true,
-              presets: ['react', 'env']
+              presets: [
+                '@babel/preset-react',
+                [
+                  '@babel/preset-env',
+                  {
+//                    targets: { browsers: ['chrome 77'] },
```

Comment:
Do we need these commented lines?

Reply:
It's just a memory help if someone (me) want to build a very optimize version.
I spent a lot of time to understand the different optimize mode.
I can remove it if you want.